### PR TITLE
[16.0][ADD] hr_employee_group_overview_readonly

### PR DESCRIPTION
**Pull request by Murtaza-OSI**

### CODE REVIEW

**Comment by SMaciasOSI** [comment]
@max3903

File: hr_employee_group_overview_readonly/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+hr_skills.access_hr_employee_skill_report,hr.employee.skill.report,hr_skills.model_hr_employee_skill_report,hr.group_hr_user,1,0,0,0
+hr_skills.access_hr_employee_skill_log,hr.employee.skill.log,hr_skills.model_hr_employee_skill_log,hr.group_hr_user,1,0,0,0
+hr_gamification.challenge_officer,"Challenge Officer",gamification.model_gamification_challenge,hr.group_hr_user,1,0,0,0
+hr_gamification.challenge_line_officer,"Challenge Line Officer",gamification.model_gamification_challenge_line,hr.group_hr_user,1,0,0,0
+hr_gamification.badge_officer,"Badge Officer",gamification.model_gamification_badge,hr.group_hr_user,1,0,0,0
+hr_gamification.badge_user_officer,"Badge-user Officer",gamification.model_gamification_badge_user,hr.group_hr_user,1,0,0,0
+access_hr_contract_history_user,hr.contract.history.user,hr_contract.model_hr_contract_history,hr.group_hr_user,1,0,0,0
+access_res_partner_hr_user,access_res_partner_hr_user,base.model_res_partner,hr.group_hr_user,1,0,0,0
+access_hr_users_employee,hr.holidays.employee.users,hr_holidays.model_hr_leave,hr.group_hr_user,1,0,0,0
+
```
Comment (max3903) [inline hr_employee_group_overview_readonly/security/ir.model.access.csv]:
```suggestion
```

File: hr_employee_group_overview_readonly/security/ir.model.access.csv [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+access_hr_resume_line__officer,hr.resume.line,hr_skills.model_hr_resume_line,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,1
+access_hr_resume_line_type_officer,hr.resume.line.type,hr_skills.model_hr_resume_line_type,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,1
+access_hr_skill_type_officer,hr.skill.type,hr_skills.model_hr_skill_type,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,1
+access_hr_skill_level_officer,hr.skill.level,hr_skills.model_hr_skill_level,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,1
+access_hr_skill_officer,hr.skill,hr_skills.model_hr_skill,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,1
+access_hr_employee_skill_officer,hr.employee.skill,hr_skills.model_hr_employee_skill,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,1
+access_hr_employee_skill_report_officer,hr.employee.skill.report,hr_skills.model_hr_employee_skill_report,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,1
+access_hr_employee_skill_log_officer,hr.employee.skill.log,hr_skills.model_hr_employee_skill_log,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,0
+
```
Comment (max3903) [inline hr_employee_group_overview_readonly/security/ir.model.access.csv]:
```suggestion
```

File: hr_employee_group_overview_readonly/security/ir.model.access.csv [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+challenge_hr_officer,"Challenge Hr Officer",gamification.model_gamification_challenge,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,1
+challenge_line_hr_officer,"Challenge Line HR Officer",gamification.model_gamification_challenge_line,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,1
+badge_hr_officer,"Badge HR Officer",gamification.model_gamification_badge,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,1
+badge_user_hr_officer,"Badge-user HR Officer",gamification.model_gamification_badge_user,hr_employee_group_overview_readonly.group_hr_officer,1,1,1,1
+
+
+
```
Comment (max3903) [inline hr_employee_group_overview_readonly/security/ir.model.access.csv]:
```suggestion
```

File: hr_employee_group_overview_readonly/views/hr_employee_views.xml [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+                position="attributes"
+            >
+                <attribute
+                    name="groups"
+                >hr_holidays.group_hr_holidays_user,hr_employee_group_overview_readonly.group_hr_overview_readonly</attribute>
+            </xpath>
+        </field>
+    </record>
+
+
```
Comment (max3903) [inline hr_employee_group_overview_readonly/views/hr_employee_views.xml:74]:
```suggestion
```

File: hr_employee_group_overview_readonly/views/hr_employee_views.xml [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+            <xpath
+                expr="//button[@name='action_open_contract_history']"
+                position="attributes"
+            >
+                <attribute name="groups">
+                    hr_contract.group_hr_contract_manager,hr_employee_group_overview_readonly.group_hr_overview_readonly</attribute>
+            </xpath>
+        </field>
+    </record>
+
```
Comment (max3903) [inline hr_employee_group_overview_readonly/views/hr_employee_views.xml:94]:
```suggestion
```

File: hr_employee_group_overview_readonly/views/hr_employee_views.xml [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+            <xpath
+                expr="//button[@name='action_open_employee_cars']"
+                position="attributes"
+            >
+                <attribute name="groups">
+                    hr_employee_group_overview_readonly.group_hr_overview_readonly,fleet.fleet_group_manager</attribute>
+            </xpath>
+        </field>
+    </record>
+
```
Comment (max3903) [inline hr_employee_group_overview_readonly/views/hr_employee_views.xml:111]:
```suggestion
```

File: hr_employee_group_overview_readonly/readme/CONTRIBUTORS.rst [hunk 7]
```diff
@@ -0,0 +1,2 @@
+* Murtaza Mithaiwala <mmithaiwala@opensourceintegrators.com>
+
```
Comment (max3903) [inline hr_employee_group_overview_readonly/readme/CONTRIBUTORS.rst]:
```suggestion
```

File: hr_employee_group_overview_readonly/views/hr_employee_views.xml [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+            <field name="coach_id" position="after">
+                <field name="total_overtime" invisible="1" />
+            </field>
+            <page name="personal_information" position="attributes">
+                <attribute name="groups">
+                    hr.group_hr_user,hr_employee_group_overview_readonly.group_hr_overview_readonly</attribute>
+            </page>
+        </field>
+    </record>
+
```
Comment (max3903) [inline hr_employee_group_overview_readonly/views/hr_employee_views.xml:20]:
```suggestion
```

**Review by max3903 — APPROVED** [review]

**Pull request merged**